getting-started/background/translate_test.go

```diff
@@ -96,3 +66,40 @@ func TestTranslate(t *testing.T) {
 		t.Fatalf("Translate got:\n%+v\nWant:\n%+v", translations[0], want)
 	}
 }
+
+func deleteAll(ctx context.Context, projectID string) error {
+	client, err := firestore.NewClient(ctx, projectID)
```

Comment:
Why create a new client on each request?

Reply:
Done (only used for tests, but might as well be more efficient).

Reply:
Oops, missed that this was in _test.go